addons/im_livechat/models/discuss_channel.py

```diff
@@ -78,6 +78,12 @@ class DiscussChannel(models.Model):
         compute="_compute_livechat_agent_providing_help_history",
         store=True,
     )
+    livechat_note = fields.Html(
+        "Livechat Note",
+        default="",
+        sanitize_style=True,
+        help="Note about the livechat channel, visible to the operator.",
```

Comment:
operator -> agent (note in this case, it's rather visible to all internal users)